includes/engelsystem_provider.php

```diff
@@ -87,4 +88,4 @@
 
 load_auth();
 
-?>
```

Comment:
You didn't add it, it was there before but if you touch the file you can remove it right away